Feature: Added tooltip to show message sender name.

iamprayush opened the pull request:
Fixes #232

- Used [ui-kitten's tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview/#tooltip).
- Upon long-pressing a message bubble, the tooltip will show the sender's name, just like in the web app.

![tooltip_demo](https://user-images.githubusercontent.com/35144226/100253186-cd273300-2f66-11eb-9867-a6c2b238b827.gif)

muhsin-k commented:
@iamprayush Thanks for the PR ❤️. Currently, we are using [UI Kitten](https://akveo.github.io/react-native-ui-kitten/) as the main UI framework. We suggest using [tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview#tooltip) provided by UI kitten. Adding additional package will increase bundle size.

iamprayush commented:
> @iamprayush Thanks for the PR . Currently, we are using [UI Kitten](https://akveo.github.io/react-native-ui-kitten/) as the main UI framework. We suggest using [tooltip](https://akveo.github.io/react-native-ui-kitten/docs/components/tooltip/overview#tooltip) provided by UI kitten. Adding additional package will increase bundle size.

@muhsin-k I've now removed react-native-elements and used the ui-kitten tooltip instead. I've also updated the demo gif, PTAL!

muhsin-k commented:
@iamprayush We will test it out in both Android and iOS and let you know if any change is required

muhsin-k commented on src/components/ChatMessageItem.js:
```diff
@@ -163,6 +175,16 @@ const ChatMessageItemComponent = ({ type, message, eva: { style, theme }, create
+      visible={tooltipVisible}
+      onBackdropPress={() => setTooltipVisible(false)}
+      placement="top">
+      <Text style={style.tooltipText}>Sent by: {message.sender.name}</Text>
```
@iamprayush The app will crash if the message sender is empty (Some messages doesn't have sender details). So show tooltip only for messages which contain sender details.

iamprayush replied:
I've added the tooltip conditionally now, PTAL!

muhsin-k reviewed (approved):
Looks good to me
